src/python/dart/engine/emr/metadata.py

```diff
@@ -14,6 +14,32 @@ class EmrActionTypes(object):
                     'description': 'The contents of this script will be executed as a bootstrap step',
                     "x-schema-form": {"type": "textarea"}
                 },
+                'configurations': {
```

Comment:
For the first point above, I could define a second level of nesting since that's all I need for my use case (if it doesn't work).